src/config.ts

```diff
@@ -45,6 +45,9 @@ class Config extends EventEmitter {
 		supervisorOfflineMode: { source: 'config.json', default: false },
 		hostname: { source: 'config.json', mutable: true },
 		persistentLogging: { source: 'config.json', default: false, mutable: true },
+		dnsServers: { source: 'config.json', default: '', mutable: true },
```

Comment:
A default of an empty string vs undefined?  Does this mean the property will be added but be an empty string?